### PR TITLE
add workflow for azure-ipi-ingress-custom-ingresscontroller

### PR DESCRIPTION
**Pull request by lihongan**
to address https://issues.redhat.com/browse/OCPQE-10821

### CODE REVIEW

**Comment by lihongan** [comment]
@jianlinliu @gpei Please help take a look. Thanks.

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ingress/custom-ingresscontroller/provision/cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource
+  - ref: ingress-azure-custom-ingresscontroller
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ingress/custom-ingresscontroller/provision/cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision-chain.yaml:16]: Sound like this is kind of post-actions for installation, if that, we can replace the above lines with `chain: cucushift-installer-rehearse-azure-ipi-provision`

Comment (lihongan) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ingress/custom-ingresscontroller/provision/cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision-chain.yaml:16]: Yes, it is post-action for installation but it needs to create new subnet so the ref `azure-provision-resourcegroup, azure-provision-vnet, ipi-conf-azure-provisionednetwork` are required, if using `chain: cucushift-installer-rehearse-azure-ipi-provision` then we cannot create new subnet in the default vnet.

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ingress/custom-ingresscontroller/provision/cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision-chain.yaml:16]: Good to know.

**Pull request merged**